enricher/standard/src/main/java/io/fabric8/maven/enricher/standard/openshift/DeploymentConfigEnricher.java

```diff
@@ -52,10 +53,17 @@ public DeploymentConfigEnricher(MavenEnricherContext context) {
 
     @Override
     public void create(PlatformMode platformMode, KubernetesListBuilder builder) {
+
         if(platformMode == PlatformMode.openshift) {
-            for(HasMetadata item : builder.buildItems()) {
 
-                if(item instanceof Deployment) {
+            boolean switchToDeployment = false;
```

Comment:
I would extract this to a method in BaseEnricher and a static constant for flag name. Since there is already `getValueFromConfig()` method available; this can make your work a bit easier

Reply:
Aah.. I made this changes in a bit of rush. Forgot about BaseEnricher and static constant.

Reply:
np